dependabot/resources/extensions.json

```diff
@@ -40,6 +40,20 @@
                 "ballerina-distribution"
             ]
         },
+        {
+            "name": "module-ballerina-ldap",
+            "level": 1,
+            "group_id": "io.ballerina.lib",
+            "artifact_id": "ldap-ballerina",
+            "version_key": "stdlibLdapVersion",
+            "default_branch": "master",
+            "auto_merge": true,
+            "push_to_central": true,
+            "is_extended_library_module": true,
```

Comment:
```suggestion
            "is_extended_library_module": false,
```

Reply:
This needs to be fixed in the master as well.